extensions/interactions/LogicProof/directives/logic_proof_short_response_directive.html

```diff
@@ -4,5 +4,5 @@
 </span>
 <span ng-if="!answer.correct">
   <span translate="I18N_INTERACTIONS_LOGIC_PROOF_INCORRECT_ANSWERS"></span>:
-  <pre><[answer.displayed_proof[1] | truncateAtFirstLine]></pre>
```

Comment:
Do we have to do something like this for `ng-if="answer.correct"`?

Reply:
I don't think so actually. It is an interesting setup, but the way it works are that correct answers are sent as strings, while incorrect answers are arrays of 3 strings where the middle value is the erroneous line. That's why truncation wasn't working, because only the line with the error was being displayed on wrong answers.

Reply:
Thanks for checking!